assets/js/base/components/cart-checkout/form-step/index.js

```diff
@@ -35,22 +35,36 @@ const FormStep = ( {
 	description,
 	children,
 	disabled = false,
+	showStepNumber = false,
```

Comment:
I see the default is still `false` - that seems the wrong way around, as most steps have the number. Not a blocker, just checking your intention here.

Reply:
Yeah... I don't have a strong opinion on this. Changed in 73974266265ea318aadb15ecd7b1bdb56e1131e2.

Thanks for the review.